app/uk/gov/hmrc/thirdpartydeveloperfrontend/connectors/ApmConnector.scala

```diff
@@ -94,7 +94,7 @@ class ApmConnector @Inject()(http: HttpClient, config: ApmConnector.Config, metr
     http.GET[ExtendedApiDefinition](s"${config.serviceBaseUrl}/combined-api-definitions/$serviceName")
 
   def fetchCombinedApi(serviceName: String)(implicit hc: HeaderCarrier): Future[Either[Throwable, CombinedApi]] =
-    http.GET[CombinedApi](s"${config.serviceBaseUrl}/combined-apis/$serviceName")
+    http.GET[CombinedApi](s"${config.serviceBaseUrl}/combined-rest-xml-apis/developer/$serviceName")
```

Comment:
This method appears to have nothing to do with a developer yet calls and endpoint with developer in the URI.

Reply:
developerId is provided as an optional query parameter